features/resource_status.md

```diff
@@ -0,0 +1,220 @@
+[//]: # (into the RFCs, which do need approval for alpha. You may find value to negotiate within the work group where the)
+[//]: # (user stories are presented to help clarify the user stories.)
+
+- [ ] [RFC Authored] - [create an RFC using template](https://docs.google.com/document/d/1ewJoCcw5-04crH-M0xw4zFxz1cfwVCPnNyW4K3m4Yyc/template/preview).
+
+[//]: # (An RFC is mandatory to graduate to experimental. The RFC does not have to be reviewed in a work group)
+[//]: # (meeting to graduate to experimental.)
+
+- [ ] [Documentation](insert_your_link_here) for enabling and using the feature.
+
+[//]: # (The documentation instructions may exist on the developer wiki or the team drive. They may include instructions)
+[//]: # (for building running a `istioctl experimental command`, or using the preview profile,)
+[//]: # (or any other relevant information.)
+
+- [ ] [Feedback plan](insert_your_link_here).
+
+[//]: # (This may include user feedback meetings, discuss.istio.io conversations, GitHub issues, or mailing lists.)
+
+- [ ] Disabled by default.
+
+- [ ] No impact on performance when the feature is disabled.
+
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Alpha
+
+### Requirements: 
+
+**Design**
+
+- [x] RFC has been approved describing the intention of the feature as well as the user stories behind the feature. 
+
+**Config**
+
+- [x] Explicit user action is required to enable this feature (e.g. a config field, config resource, or installation action). 
+
+> Link to instructions for enabling
+    https://istio.io/latest/docs/ops/diagnostic-tools/istioctl-analyze/
+
+**Docs**
+
+- [x] Reference docs are published to preliminary.istio.io or the Istio wiki.
+- [x] Basic feature docs are published on preliminary.istio.io describing what the feature does, how to use it, and any caveats. 
+- [x] Release notes entries added as appropriate
+- [x] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [x] Automated integration tests cover core use cases with the feature enabled. 
+- [x] When disabled, the feature does not affect system stability or performance. 
+
+**API**
+
+- [x] Initial API review.
+
+**Approvals**
+
+- [x] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [x] The TOC has reviewed and approved promotion of the feature as part of the
+	roadmap for a release.
+
+**Promotion**
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+
+---
+
+## Beta
+
+### Requirements: 
+
+**Design**
+
+- [x] Design doc describing the intention of the feature, how it will be
```

Comment:
Can you include a link?

Reply:
https://docs.google.com/document/d/1OipeJ1NnBAGkw27CSL02eykbhF8zRdWKBfwtrqQOhNE/edit#heading=h.qex63c29z2to

@howardjohn this is the doc that was generated long time ago, I did some updates recently.

Reply:
The link is also in this PR.

Reply:
That is not a proper design doc, that only describes the schema. My concerns are not with the schema but everything else

Reply:
@howardjohn john, you would like to see a document with some design on how this should been implemented, right? like when it will start to run, how analyzer is designed, how an analyzer gets invoked, resources being retrieved etc, are these the things that you are looking for?